### ISSUE TITLE
[DOC] Add Internationalization (i18n) Support to Longhorn Documentation

### ISSUE THREAD

**Issue by samnoonabrar** [longhorn/longhorn#10827]
## Add Internationalization (i18n) Support to improve global usability

The Longhorn documentation currently lacks internationalization (i18n) support, limiting accessibility for non-English speaking users. This issue proposes adding i18n support to enable translation of the documentation into multiple languages, improving global usability.

## Goals

* Support at least one additional language (e.g., Spanish, Chinese, or French) as a proof of concept.

* Ensure maintainability of translations and scalability for adding more languages.

* Minimize disruption to the existing documentation workflow.

**Comment by derekbit** [comment]
cc @sushant-suse

**Comment by derekbit** [comment]
cc @sushant-suse @jbrockmeyer @innobead
